packages/firebase_data_connect/firebase_data_connect/lib/src/common/common_library.dart

```diff
@@ -15,7 +15,7 @@ part 'dataconnect_options.dart';
 enum CallerSDKType { core, generated }
 
 String getGoogApiVal(CallerSDKType sdkType, String packageVersion) {
-  String apiClientValue = 'gl-dart/flutter fire/$packageVersion';
+  String apiClientValue = 'gl-dart/$packageVersion fire/$packageVersion';
```

Comment:
the one after gl-dart/ is supposed to be dart language version. So it should be gl-dart/DART_VERSION. 

If we can't get dart version lets leave it blank like we do with gl-js/

Reply:
https://github.com/firebase/data-connect-ios-sdk/blob/main/Sources/Internal/GrpcClient.swift#L84

Reply:
Looks like for flutter we just repeat the packageVersion: https://github.com/firebase/flutterfire/blob/ff1e5f672cee29731dc4d21251611030add9e605/packages/firebase_vertexai/firebase_vertexai/lib/src/vertex_model.dart#L97

Reply:
Sgtm then.